samples/hello_world/boards/qemu_cortex_m0.conf

```diff
@@ -0,0 +1,3 @@
+# Make this as small an example as possible
+CONFIG_MULTITHREADING=n
```

Comment:
Do not want, drivers are mostly useless without multithreading, what use is this?

Reply:
MCUboot is one example where thread support may not be needed and configuring it out saves a lot relatively speaking.

Reply:
Yes, and it's disabled by default for mcuboot (which is a special case), so what use is that to an application that is going to be doing something, most likely with external hardware?

Reply:
Everyone's app is its own special case. If there is justification for disablind thread support so Zephyr could be used in an extremely constrained case then why would you be against that? You are not forced to disable threads in your own project if you don't want to.

Reply:
> If there is justification for disablind thread support so Zephyr could be used in an extremely constrained case then why would you be against that

No, and as Carles said, that's why there is a "minimal" sample to demonstrate that